packages/algob/src/internal/deployer.ts

```diff
@@ -157,12 +158,21 @@ class DeployerBasicMode {
     return this.checkpoint.getAppfromCPKey(appName);
   }
```

Comment:
let's update doc string and guide for `getApp` function: what's happening if someone will deploy two same smart contracts (same nameApproval and nameClearr)? What that  function will return? Maybe we should deprecate it? Or do a breaking change and use name only?

Reply:
@vuvth what do you think?

Reply:
needs more discussion (breaking change)